src/imarker_robot_state.cpp

```diff
@@ -97,7 +98,9 @@ IMarkerRobotState::IMarkerRobotState(planning_scene_monitor::PlanningSceneMonito
     else
       eef_name = imarker_name + "_left";
 
-    end_effectors_[i] = std::make_shared<IMarkerEndEffector>(this, eef_name, arm_datas_[i], color);
+    // respect Eigen alignment
+    end_effectors_[i] = std::allocate_shared<IMarkerEndEffector>(Eigen::aligned_allocator<IMarkerEndEffector>(), this,
```

Comment:
Having `IMarkerEndEffector` declaring `EIGEN_MAKE_ALIGNED_OPERATOR_NEW`, why do you need this change here as well?
Found the answer myself: https://gitlab.com/libeigen/eigen/issues/1049